pages/nyheter/[slug].tsx

```diff
@@ -30,13 +47,17 @@ const BlogTemplate = ({ article }: Props) => {
         duration: 0.5
       }}
     >
-      <article>
+      <Article>
         <Header>{article.data.title}</Header>
         <div>
           <ReactMarkdown source={article.content} />
         </div>
-        <a href="/nyheter">tilbake</a>
-      </article>
+      </Article>
+      <TopLeft>
+        <Link href="/nyheter">
+          <FunkyArrow color={colors.lightBlue}>{"<------"}</FunkyArrow>
```

Comment:
Is it possible to just include the `<------` in the FunkyArrow component itself?